packages/xrpl/src/sugar/autofill.ts

```diff
@@ -3,15 +3,22 @@ import { xAddressToClassicAddress, isValidXAddress } from 'ripple-address-codec'
 
 import type { Client } from '..'
 import { ValidationError, XrplError } from '../errors'
-import { AccountInfoRequest, AccountObjectsRequest } from '../models/methods'
+import {
+  AccountInfoRequest,
+  AccountObjectsRequest,
+  ServerInfoResponse,
+} from '../models/methods'
 import { Transaction } from '../models/transactions'
 import { setTransactionFlagsToNumber } from '../models/utils/flags'
-import { xrpToDrops } from '../utils'
+import { isEarlierRippledVersion, xrpToDrops } from '../utils'
 
 import getFeeXrp from './getFeeXrp'
 
 // Expire unconfirmed transactions after 20 ledger versions, approximately 1 minute, by default
 const LEDGER_OFFSET = 20
+const RESTRICTED_NETWORKS = 1024
```

Comment:
I think it'd be useful to add a comment explaining what restrited networks means since I wouldn't know just from reading the name Ex. 
```suggestion
// Sidechains are expected to have network IDs above this. 
// Mainnet and testnet are exceptions. More context: https://github.com/XRPLF/rippled/pull/4370
const RESTRICTED_NETWORKS = 1024
```

Reply:
I feel the comment is still unclear. Please explain what "restricted" means in the comment explicitly.